src/convertToObject.js

```diff
@@ -5,8 +5,26 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  const stylesArray = stylesString.split(';');
```

Comment:
[CODE STYLE] - Avoid variable namings like `stylesArray`. Descriptive names improve code readability. Consider a name that reflects the content of the array, such as `styleDeclarations`.